openapi/generator.go

```diff
@@ -877,6 +877,13 @@ func (g *Generator) newSchemaFromType(t reflect.Type) *SchemaOrRef {
 		nullable = true
 	}
 	dt := g.datatype(t)
+
+	// Check for "Nullable"
```

Comment:
Redundant/not useful comment.

Also, the implementation should first use `reflect.Type.Implements()` to check if the type implements the interface before creating a new value with `reflect.New` to call the `Nullable` method.

You can know the type of an interface like so:
```
tofNullable = reflect.TypeOf((*Nullable)(nil)).Elem()
```

ex: https://github.com/wI2L/fizz/blob/master/openapi/types.go#L15

which is then used like so:
```
t.Implements(tofNullable)
```

https://github.com/wI2L/fizz/blob/master/openapi/types.go#L123

Reply:
Makes sense! Thanks 👍